x-pack/plugins/spaces/server/lib/request_interceptors/on_request_interceptor.ts

```diff
@@ -12,8 +12,8 @@ import {
 import { format } from 'url';
 import { DEFAULT_SPACE_ID } from '../../../common/constants';
 import { modifyUrl } from '../utils/url';
-import { LegacyAPI } from '../../new_platform/plugin';
-import { getSpaceIdFromPath } from '../../../common';
+import { LegacyAPI } from '../../plugin';
+import { getSpaceIdFromPath } from '../../../common/lib/spaces_url_parser';
```

Comment:
__nit:__ I guess you could just leave `../../../common` as it was before?

Reply:
Yep, thanks! This was a leftover change from when the `common/index` referenced code that could never run on the server since it relied on `window`. That was since moved into the `public` directory, but I never updated these references.